platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -87,6 +87,8 @@
                 dialog.getModalBody().css('padding', '0');
 
                 dialog.open();
+
+                dialog.getModal().prev().css('z-index', dialog.getModal().css('z-index'));
```

Comment:
So it doesn't use our own ModalService? If so, it probably should and it should also fix the problem.